### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by madhuBayy**
@ldko @somexpert 
Some tests using current time are changed to avoid DST issues and flake8 errors are taken care of in this PR. Please review the changes.

### CODE REVIEW

File: codalib/util.py [hunk 1]
```diff
@@ -294,7 +294,7 @@ def updateQueue(destinationRoot, queueDict, debug=False):
         print uploadXMLText
     try:
         response, content = doWebRequest(url, "PUT", data=uploadXMLText)
-    except:
+    except (urllib2.URLError, urllib2.HTTPError):
```
Comment (ldko) [inline codalib/util.py]: Since `urllib2.HTTPError` is a subclass of `urllib2.URLError`, it should be the same to use:
    `except urllib2.URLError:`

**Review by ldko — COMMENTED** [review]
One minor comment, but works for me!

**Review by ldko — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -2,13 +2,9 @@ language: python
 sudo: false
 python: 2.7
 env:
-    - TOX_ENV=py26
     - TOX_ENV=py27
-    - TOX_ENV=py34
     - TOX_ENV=flake8
```
Comment (somexpert) [inline .travis.yml]: I can't see a `flake8` env now without suggesting we add the python version specifier :)

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ A helper library for [coda](https://github.com/unt-libraries/coda)
 
 ## System Requirements
 
-* python-dev (Python 2.6+)
+* python-dev (Python 2.7+)
```
Comment (somexpert) [inline README.md]: Should probably remove the '+' so we don't give the impression that it's currently compatible with Python 3.
```suggestion
* python-dev (Python 2.7)
```

**Review by somexpert — COMMENTED** [review]
Got a couple small recommendations, aside from us also dealing with that odd commit with the future date.

**Review by somexpert — APPROVED** [review]

**Pull request merged**